.ci/magic-modules/create-pr.sh

```diff
@@ -51,6 +52,7 @@ if [ "$BRANCH_NAME" = "$ORIGINAL_PR_BRANCH" ]; then
       git checkout -b "$BRANCH_NAME"
       if TF_PR=$(hub pull-request -b "$TERRAFORM_REPO_USER/$PROVIDER_NAME:master" -F ./downstream_body); then
         DEPENDENCIES="${DEPENDENCIES}depends: $TF_PR ${NEWLINE}"
+        LABELS="${LABELS}${PROVIDER_NAME},"
```

Comment:
`$PROVIDER_NAME` is `terraform-provider-google-beta` - you want `$SUBMODULE_DIR`, which is `google-beta`.

Reply:
Don't we want the label to be "terraform-provider-google-beta" and not "google-beta"?